### ISSUE TITLE
docs is not offline

### ISSUE THREAD

**Issue by HuStmpHrrr** [jekyll/jekyll-docs#13]
it turns out some links in docs are still pointing to the website https://jekyllrb.com

for example, in my local docs,

docs -> Installation points to https://jekyllrb.com/docs/installation/

``` html
       <option value="https://jekyllrb.com/docs/installation/">
        Installation
       </option>
```

**Comment by HuStmpHrrr** [comment]
mentioning that, is that possible to have the paths to be relative? it's gonna be easier for me to build up a apache to host different docs in different paths.

for instance, in this case, instead of a abs path `/docs/installation/`, wouldn't `../../docs/installation/` better(if it's a button somewhere in the docs page)?

**Comment by bryanjhv** [comment]
Tha's merely a problem of Jekyll docs site [configuration](https://github.com/jekyll/jekyll/blob/v3.0.0/site/_config.yml#L18) that has been fixed with Jekyll 3.3.0, as you can see in jekyll/jekyll#5431.
